IT translation improvements

ironbishop opened the pull request.

ironbishop committed:
IT translation improvements

jancborchardt commented on index.html:
```diff
@@ -69,7 +69,7 @@ <h3 id="participate" class="translatable">Participate</h3>
 		</div>
 		<div class="tosdr">
-			<p class="translatable label" style="display:block"><abbr title='Terms of service ratings provided by https://tosdr.org'>TOS rating</abbr></p>
+			<p class="label" style="display:block"><abbr class="translatable" title='Terms of service ratings provided by https://tosdr.org'>TOS rating</abbr></p>
```
Shouldn’t the class be on both elements? In any case, it would be good if both the content "TOS rating" and the title would be translated. :)

yuwash replied:
It looks like no matter where you put the class, it would only apply to the content and not for the `title` attribute. See [comment below](#issuecomment-307375139)

ironbishop commented:
Any idea on how to translate the tooltip, too?

yuwash commented:
For applying translation to the `title` attribute the part [js/default.js#L131](https://github.com/libreprojects/libreprojects/blob/gh-pages/js/default.js#L131) would need to change; Maybe introduce another (something like) `title-translatable` class and do another translation routine for that

ironbishop committed:
tooltip translations

ironbishop committed:
tooltip for language selection

ironbishop commented:
I added a check because of a null pointer exception, and borrowed tooltip descriptions from the french locale. I updated the ABBR tag there, too.

I added a tooltip for the locale changer, because the icons are so small, I thought it can be useful (expecially because FR and IT flags are similar, small, and near each others).

ironbishop committed:
fix 7ba3242

ironbishop commented:
Closing for lack of interest, and i honestly dont remember what i wrote myself after all this time.
